compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -888,6 +888,12 @@ pub static BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         EncodeCrossCrate::No, loop_match, experimental!(loop_match)
     ),
 
+    gated!(
+        transparent, Normal,
+        template!(Word, "https://github.com/rust-lang/rust/issues/79260#issuecomment-731773625"),
```

Comment:
probably also want to update this to point to actual documentation (I don't intend to add that, if its a blocker then leave this open or close it until someone comes along who wants to push this over the finish line)